api/api.go

```diff
@@ -100,6 +101,14 @@ func (api *ManagementAPI) DeleteRecording(w http.ResponseWriter, r *http.Request
 	io.WriteString(w, "cptv file deleted")
 }
 
+func (api *ManagementAPI) TakeSnapshot(w http.ResponseWriter, r *http.Request) {
+	_, err := exec.Command("sh", "-c", "dbus-send --system --type=method_call --print-reply --dest=org.cacophony.thermalrecorder /org/cacophony/thermalrecorder org.cacophony.thermalrecorder.TakeSnapshot").Output()
```

Comment:
Spawning a process to make a dbus request isn't a great approach, especially for something that could potentially get called rapidly. Better to pull in godbus and use it to make the call. 

Something like (untested):
```go
conn, err := dbus.SystemBus()
recorder := conn.Object("org.cacophony.thermalrecorder", "/org/cacophony/thermalrecorder")
err := recorder.Call("org.cacophony.thermalrecorder.TakeSnapshot")
...
```